config/crd/bases/dataplane.openstack.org_openstackdataplanenodes.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.10.0
+    controller-gen.kubebuilder.io/version: v0.9.2
```

Comment:
it is reverting the kubebuilder version, please remove your `./bin/controller-gen` so you can get an updated one

Reply:
Done.